src/test/java/com/automatedtest/sample/definitions/SearchImageSteps.java

```diff
@@ -0,0 +1,23 @@
+package com.automatedtest.sample.definitions;
+
+import com.automatedtest.sample.pages.SearchResultPage;
+import io.cucumber.java.en.And;
+import io.cucumber.java.en.When;
+
+public class SearchImageSteps {
+    private final SearchResultPage searchResultPage;
+
+    public SearchImageSteps() {
+        this.searchResultPage = new SearchResultPage();
+    }
+
+    @And("I change to image tab")
```

Comment:
`@And` is just a circumstance keyword that stand for Given, When, Then. We should only defy a step as Given, When or Then.

Reply:
please fix for other and keywords as well